src/Akka.Streams.Kafka/Settings/Subscriptions.cs

```diff
@@ -7,10 +7,32 @@
 namespace Akka.Streams.Kafka.Settings
 {
     public interface ISubscription { }
-    public interface IManualSubscription : ISubscription { }
+
+    public interface IManualSubscription : ISubscription
+    {
+        /// <summary>
+        /// Statistics handler
+        /// </summary>
+        Option<IStatisticsHandler> StatisticsHandler { get; }
+
+        /// <summary>
+        /// Allows to specify custom statistics handler. See more at <see cref="IStatisticsHandler"/>
+        /// </summary>
+        IManualSubscription WithStatisticsHandler(IStatisticsHandler statisticsHandler);
+    }
 
     public interface IAutoSubscription : ISubscription
     {
+        /// <summary>
+        /// Statistics handler
+        /// </summary>
+        Option<IStatisticsHandler> StatisticsHandler { get; }
```

Comment:
Since this handler and `WithStatisticsHandler` are members of both `IManualSubscription` and `IAutoSubscription` - do you think it could make sense to make them both members if `ISubscription` interface?

Reply:
Yes make sense to avoid code duplication, I will provide an update